content/doc/book/platform-information/support-policy-java.adoc

```diff
@@ -3,37 +3,45 @@ layout: subsection
 title:  Java Support Policy
 ---
 
-There are separate run and job execution requirements for Jenkins installations.
+There are separate runtime and job execution requirements for Jenkins installations.
 
 ## Running Jenkins system
 
-If you are installing or upgrading to Jenkins 2.357 or later, or to LTS 2.361.1 or later, Java 11 or 17 is required to run Jenkins.
-link:/blog/2022/06/28/require-java-11/[Read more about this in the announcement blog post.]
+The following Java versions are required to run Jenkins:
 
-[NOTE]
-.Upgrading Java version to 11
-====
-Upgrading an existing Jenkins setup from Java 8 to Java 11?
-Refer to the link:/doc/administration/requirements/upgrade-java-guidelines[upgrade guidelines].
-====
-
-## Running Jenkins on Java 11 in Docker
+|===
+|Supported Java versions|Long term support (LTS) release|Weekly release
 
-Java 11 Docker installation instructions are included in link:/doc/book/installing/docker/#downloading-and-running-jenkins-in-docker["Downloading and running Jenkins in Docker"].
-Alternatively, the `jenkins/jenkins:jdk17` Docker image allows you to run the Jenkins controller on Java 17.
+|Java 11, Java 17, or Java 21|N/A|2.419 (August 2023)
+|Java 11 or Java 17|2.361.1 (September 2022)|2.357 (June 2022)
+|Java 8, Java 11, or Java 17|2.346.1 (June 2022)|2.340 (March 2022)
+|Java 8 or Java 11|2.164.1 (March 2019)|2.164 (February 2019)
+|===
 
 [IMPORTANT]
 .Supported Java versions
 ====
-Only Java 11 and Java 17 are supported. If you install an unsupported Java version, your Jenkins controller will not run. 
+If you install an unsupported Java version, your Jenkins controller will not run.
 ====
 
 These requirements apply to all components of the Jenkins system, including the Jenkins controller, all types of agents, CLI clients, and other components.
+You do _not_ need to build your application with the same version of Java used to run Jenkins itself;
+see the "Running Java-based tools and builds on Jenkins" section below.
+
+[NOTE]
+.Upgrading Java to a newer version
+====
+Upgrading an existing Jenkins setup to a newer version of Java?
+Refer to the link:/doc/administration/requirements/upgrade-java-guidelines[upgrade guidelines].
```

Comment:
We have two Java upgrade guides, one for the upgrade from Java 8 to Java 11 and another for the upgrade from Java 11 to Java 17.  I think that we should include a link to both of those upgrade guides, in case users need to refer first to the Java 8 to Java 11 document, then to the Java 11 to java 17 document

We prefer to point to the redirect destination rather than the redirect source from inside the documentation so that the reader does not see a redirect when clicking a link in the Jenkins documentation pages.  
```suggestion
Refer to the link:/doc/book/platform-information/upgrade-java-to-17[Java 11 to 17 upgrade guidelines] and the link:/doc/book/platform-information/upgrade-java-to-11[Java 8 to 11 upgrade guidelines].
```